setup.cfg

```diff
@@ -38,9 +38,9 @@ install_requires =
 setup_requires =
   setuptools_scm
 
+[options.package_data]
+labscript_profile = ../labscript-suite.pth
```

Comment:
How does this relate to #43?

Reply:
The issue is where the file is on disk, so whether it's package_data or something else I don't think is super relevant. Pip doesn't care about it, it's just adding the directory your importable modules are in to the path and making an .egg-info file. (also, the file was package_data before this change, I just moved it into the .cfg since it no longer needs to be behind an `if` statement).